tests/fixtures/odl.py

```diff
@@ -65,7 +66,7 @@ class ODLTestFixture:
     def __init__(
         self,
         db: DatabaseTransactionFixture,
-        files: ODLAPIFilesFixture,
+        files: APIFilesFixture,
```

Comment:
What happened here?

Reply:
Without this change, mypy gives this error: 
```
tests/fixtures/odl.py:296: error: Argument 2 to "ODL2TestFixture" has incompatible type "ODL2APIFilesFixture"; expected "ODLAPIFilesFixture"  [arg-type]
```

`ODL2TestFixture`:
https://github.com/ThePalaceProject/circulation/blob/5ecedbc042927e219a4faab3a730732756fff19c/tests/fixtures/odl.py#L285-L286

I think this is the correct change, as both `ODLAPIFilesFixture` and `ODL2APIFilesFixture` use `APIFilesFixture` as their base class.